src/ui/RealmBrowser/index.tsx

```diff
@@ -706,9 +731,17 @@ class RealmBrowserContainer
     return Object.keys(objectSchema.properties).map(propertyName => {
       const property = objectSchema.properties[propertyName];
       if (typeof property === 'object') {
+        let isEmbedded = false;
+        if (property.objectType) {
+          const propertyObjectSchema = this.realm?.schema.find(schema => {
+            return schema.name === property.objectType;
+          });
+          isEmbedded = propertyObjectSchema?.embedded ?? false;
+        }
```

Comment:
Consider simplifying this to two `const`s

Reply:
Done... using `isEmbeddedType`.